src/System.Private.CoreLib/shared/System/Reflection/TypeDelegator.cs

```diff
@@ -122,9 +123,9 @@ public TypeDelegator(Type delegatingType)
 
         // ICustomAttributeProvider
         public override object[] GetCustomAttributes(bool inherit) => typeImpl.GetCustomAttributes(inherit);
-        public override object[] GetCustomAttributes(Type attributeType, bool inherit) => typeImpl.GetCustomAttributes(attributeType, inherit);
+        public override object[] GetCustomAttributes(Type? attributeType, bool inherit) => typeImpl.GetCustomAttributes(attributeType, inherit);
```

Comment:
Github won't let me comment in it, but I think `GetElementType` can return `null` -- based on some implementations of `Type` it can return `null`. Actually, `HasElementType` returns, `_elementType != null`.